paid-courses.md

```diff
@@ -5,6 +5,8 @@ We list [free sites on the main page](README.md), while these great sites requir
 * [YouthDigital](http://www.youthdigital.com): Learn Design, Code & Animate
 * [Team Treehouse](https://teamtreehouse.com)
 * [Tynker](https://www.tynker.com)
+* [Kano Computer & Coding Kits](https://kano.me)
```

Comment:
I know the free/paid is a little bit of a mess, but (to me) this would fit much better, and more prominently, in that Hardware > Kits section.. do you mind moving it there? PS: Also, if this is your first Pull Request ever (looks like so, at least on GitHub), it would be no fun if it just got merged without a real review... :smile: